### PR TITLE
[#166240800] allow user to report an article

### PR DESCRIPTION
**Pull request by Ramadhan0**
#### What does this PR do?
 * Enable  users to report article
#### Description of Task to be completed?
 *  enable user to report an article
#### How should this be manually tested?
* Clone the repo to the local machine.
* Run ``npm install``   to install the dependencies.
* Set up the   ``.env`` as instructed by the   ``.env-example``
* Run  ``npm dev``.`.
*  Navigate to ``/articles``.

### [166240800](https://www.pivotaltracker.com/story/show/166240800)
Screen Shot 2019-09-12 at 14 44 47

Screen Shot 2019-09-12 at 14 44 54

Screen Shot 2019-09-12 at 15 02 34

### CODE REVIEW

File: src/App/Components/ReportArticle/ReportArticle.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      this.props.reportArticle(slug, reason, token);
+    }
+  }
+
+  render() {
+    const { isAuthor } = this.props;
+    const { hovered } = this.state;
+    const { reportDataError, reportData } = this.props;
+    const reportButton = isAuthor ? 'notAuthor' : 'report';
+    const report = (reportDataError && 'Sorry, You can not report this Article twice with the same reason!!!') || (reportData && 'Article reported successfully') || '';
```
Comment (MemunaHaruna) [inline src/App/Components/ReportArticle/ReportArticle.js]: @Ramadhan0 doesn't the `reportDataError` have the appropriate error message? Why are you hardcoding the text `'Sorry, You can not report this Article twice with the same reason!!!'`

File: src/Style/Sass/component/_report_article.scss [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+}
+
+.error,
+.success {
+  text-align: center;
+  width: 300px;
+  margin-top: 1%;
+  margin-left: 6%;
+
+  @media only screen and (max-width: 1024px) {}
```
Comment (MemunaHaruna) [inline src/Style/Sass/component/_report_article.scss]: @Ramadhan0 removed used css

File: src/Style/Sass/main.scss [hunk 3]
```diff
@@ -1,7 +1,6 @@
 @import "abstract/variables";
 @import "abstract/common";
 @import "abstract/typography";
-@import "abstract/variables";
```
Comment (MemunaHaruna) [inline src/Style/Sass/main.scss:4]: @Ramadhan0 why did you remove this?

Comment (Ramadhan0) [reply src/Style/Sass/main.scss:4]: it was imported twice @MemunaHaruna

File: src/App/Components/ReportArticle/ReportArticle.js (hunk 1 above)
Comment (Ramadhan0) [reply src/App/Components/ReportArticle/ReportArticle.js]: @MemunaHaruna  the error success messages from database contain `slug`. and i don't it is good idea to display slug on the user's screen.

**Review by MemunaHaruna — APPROVED** [review]
